bluemira/builders/EUDEMO/reactor.py

```diff
@@ -290,109 +265,88 @@ def build_in_vessel_component_shapes(self, component_tree: Component):
+        self.register_builder(builder)
+        component = super()._build_stage()
 
         return component
 
+    @Reactor.design_stage(BLANKET)
     def build_blanket(self, component_tree: Component, blanket_face):
         """
         Run the breeding blanket build.
         """
-        name = EUDEMOReactor.BLANKET
-
-        bluemira_print(f"Starting design stage: {name}")
-
         default_config = {}
-        config = self._process_design_stage_config(name, default_config)
+        config = self._process_design_stage_config(default_config)
 
         builder = BlanketBuilder(
             self._params.to_dict(), config, blanket_silhouette=blanket_face
         )
-        self.register_builder(builder, name)
-        component = super()._build_stage(name)
-
-        bluemira_print(f"Completed design stage: {name}")
+        self.register_builder(builder)
+        component = super()._build_stage()
 
         return component
 
+    @Reactor.design_stage(CRYOSTAT)
     def build_cryostat(self, component_tree: Component):
         """
         Run the cryostat vacuum vessel build.
         """
-        name = EUDEMOReactor.CRYOSTAT
-
-        bluemira_print(f"Starting design stage: {name}")
-
         thermal_shield = component_tree.get_component(
             EUDEMOReactor.THERMAL_SHIELD
         ).get_component("xz")
         cts = thermal_shield.get_component("Cryostat TS").shape.boundary[0]
 
         default_config = {}
-        config = self._process_design_stage_config(name, default_config)
+        config = self._process_design_stage_config(default_config)
 
         builder = CryostatBuilder(self._params.to_dict(), config, cts_xz=cts)
-        self.register_builder(builder, name)
-        component = super()._build_stage(name)
-
-        bluemira_print(f"Completed design stage: {name}")
+        self.register_builder(builder)
+        component = super()._build_stage()
 
         return component
 
+    @Reactor.design_stage(RADIATION_SHIELD)
     def build_radiation_shield(self, component_tree: Component):
         """
         Run the radiation shield build.
         """
-        name = EUDEMOReactor.RADIATION_SHIELD
-
-        bluemira_print(f"Starting design stage: {name}")
-
         cryostat = component_tree.get_component(EUDEMOReactor.CRYOSTAT).get_component(
             "xz"
         )
         cryo_vv_xz = cryostat.get_component("Cryostat VV").shape
 
         default_config = {}
-        config = self._process_design_stage_config(name, default_config)
+        config = self._process_design_stage_config(default_config)
 
         builder = RadiationShieldBuilder(
             self._params.to_dict(),
             config,
             cryo_vv_xz=cryo_vv_xz,
         )
-        self.register_builder(builder, name)
-        component = super()._build_stage(name)
-
-        bluemira_print(f"Completed design stage: {name}")
+        self.register_builder(builder)
+        component = super()._build_stage()
```

Comment:
Considered just directly returning `super()._build_stage()` but I dont know whether we want to make the flow and naming conventions obvious so I left it here. 

If we want to do that let me know

Reply:
I see what you mean, but I don't think that `_build_stage` necessarily tells us any more about side effects than `_build_stage_component`. Maybe the function name being less specific gives us some idea that something else is going on...

I agree, it's probably best to keep it as one function, you don't want inconsistent state. Like I said, it doesn't _really_ matter on a private function like this. It was more a reply to James's note than a real gripe.

Reply:
I agree on the code smell I will directly change to

`return super()._build_stage()`